services/event/event_service.go

```diff
@@ -162,6 +184,27 @@ func (service EventService) Update(eventRequest entities.EventRequest, id int, t
 		}
 		event.DatetimeEvent = datetime
 	}
+	if cover != nil {
+
+		// Delete previous cover
+		if event.Cover != "" {
+			u, _ := url.Parse(event.Cover)
+			objectPathS3 := strings.TrimPrefix(u.Path, "/")
+			helpers.DeleteFromS3(objectPathS3)
+		}
+
+		coverFile, err := cover.Open()
+		if err != nil {
+			return entities.EventResponse{}, web.WebError{Code: 500, Message: "cannot read cover image file"}
+		}
+		// Upload cover to S3
+		filename := uuid.New().String() + cover.Filename
+		coverURL, err := helpers.UploadFileToS3("cover/"+filename, coverFile)
```

Comment:
Saat update event, file cover yang baru di upload ke path yang tidak sama seperti di create event
```suggestion
		coverURL, err := helpers.UploadFileToS3("event/cover/"+filename, coverFile)
```